source/direct_debit/index.html.md.erb

```diff
@@ -257,23 +256,31 @@ This would return payments 501 to 1000.
     </tr>
     <tr>
     <td nowrap><code>started</code></td>
-    <td>The user has visited the <code>next_url</code> and landed on the GOV.UK Pay page to enter their account information.</td>
+    <td>Your user has visited the <code>next_url</code> and landed on the GOV.UK Pay page to enter their account information.</td>
+    </tr>
+    <tr>
+    <td nowrap><code>abandoned</code></td>
+    <td>GOV.UK Pay did not set up the mandate, because your user cancelled or took longer than 90 minutes to finish entering their account information.</td>
+    </tr>
+    <tr>
+    <td nowrap><code>failed</code></td>
+    <td>GoCardless could not set up the mandate, for example because your user's account information was incorrect or your user's bank account does not support Direct Debit.</td>
```

Comment:
i'd prefer not to use 'account information was incorrect' - because the user's bank account is checked before the details are submitted. it's a modulus check, so there's a very slim chance that incorrect details could get through. worried this might invite some questions about the robustness of our checking. 

is the one example enough?

Reply:
Thanks Katie - one example definitely works. I've amended in https://github.com/alphagov/pay-tech-docs/pull/306/commits/7ad6f07c2dc6e261a7724b1f8e665c2180950b91.